### DISCUSSION TITLE
What is _WebToolingArtifacts?

### DISCUSSION THREAD

**Discussion by Deilan** [dotnet/msbuild#6642] [Q&A] [answered]
Recently I have found `_WebToolingArtifacts` item in `.csproj` file, which looks like this:

```xml
<ItemGroup>
  <_WebToolingArtifacts Remove="Properties\PublishProfiles\FolderProfile.pubxml" />
</ItemGroup>
```

I tried to find any info about it on Google, but there's no info.

Can you please explain what is it, what is it used for, is there any documentation about it?

**Comment by rainersigwald** [comment]
@vijayrkn, can you help or route?

**Comment by vijayrkn** [comment, marked as answer]
@Deilan - Thank you for reporting this. This entry shouldn't be required in the csproj. Please feel free to remove this. 

Context: This glob is defined in the WebSDK - https://github.com/dotnet/sdk/blob/c53c8ac81e1bcb8118c31bc4f5622396c72e9959/src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.DefaultItems.props

This was introduced to ensure that the tooling artifacts are not part of the application's published output.  Since the behavior is already defined in the SDK, this should not have showed up in the csproj. Do you happen to remember what action added the following to your project file?

**Reply by haldiggs** [reply]
just had this still happening when I made a copy (ctrl-c, ctrl-v) of one of the files.

**Comment by stuartd** [comment]
In my case, renaming publish profiles - by editing the file names - resulted in this section appearing in the .csproj.
